orttraining/orttraining/core/graph/allreduce_optimizer_graph_builder.cc

```diff
@@ -96,7 +96,8 @@ static Status AddNcclAllReduceForGradients(
     GraphAugmenter::GraphDefs& graph_defs,
     ArgDef& fused_allreduce_output) {
   fused_allreduce_output = ArgDef(fused_gradient_argdef.name + "AllReduce_Out", fused_gradient_argdef.type_proto);
-
+  bool keep_looping = false;
+  while(keep_looping);
```

Comment:
debug code?

Reply:
I will remove it before checking-in, I use this to break when using multiple GPUs.

---
In reply to: [453882146](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r453882146) [](ancestors = 453882146)